### PR TITLE
Created Discord Documentation

### PR DESCRIPTION
**Pull request by erin2722**

### CODE REVIEW

File: docs/nodes/nodes-library/nodes/Discord/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+The start node exists by default when you create a new workflow.
+
+### Discord Node
+
+1. In order to create a Discord webhook that sends content to your channel, go to the settings of your channel.
+2. Click Webhooks.
+3. Name your bot and copy the link of the webhook.
+4. Paste your webhook into the *Webhook URL* field of the Discord Node.
+5. Enter your message in the *Text* field.
+6. Click on *Exceute Node* to run the workflow. :tada:
```
Comment (ivov) [inline docs/nodes/nodes-library/nodes/Discord/README.md]: Typo: `Exceute`

File: docs/nodes/nodes-library/nodes/Discord/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+## Basic Operations
+
+- Send messages in a Discord Channel
+
+## Example Usage
+
+This workflow allows you to send a message to a Discord channel using webhooks. You can also find the [workflow](https://n8n.io/integrations/n8n-nodes-base.discord) on this website. This example usage workflow would use the following two nodes.
+- [Start]()
+- [Discord]()
```
Comment (ivov) [inline docs/nodes/nodes-library/nodes/Discord/README.md]: Two links missing? But I'm not sure what they should be pointing to. Maybe the first to a future `nodes-library/core-nodes/Start/README.md` and the second to itself? (The latter wouldn't make much sense.)

Comment (ivov) [reply docs/nodes/nodes-library/nodes/Discord/README.md]: Tanay just confirmed the first link should point to `docs/nodes/nodes-library/core-nodes/Start/README.md` and the second one should _not_ be a link.

Comment (erin2722) [reply docs/nodes/nodes-library/nodes/Discord/README.md]: Sounds good! Just fixed!

File: docs/nodes/nodes-library/nodes/Discord/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+The Discord Node does not require Authentication, but you must have access to a channel's settings to use webhooks.
+
+## Basic Operations
+
+- Send messages in a Discord Channel
+
+## Example Usage
+
+This workflow allows you to send a message to a Discord channel using webhooks. You can also find the [workflow](https://n8n.io/integrations/n8n-nodes-base.discord) on this website. This example usage workflow would use the following two nodes.
+- [Start](nodes-library/core-nodes/Start/README.md)
```
Comment (ivov) [inline docs/nodes/nodes-library/nodes/Discord/README.md]: I think the link to the future Start node readme will be broken. This Discord readme will be in `docs/nodes/nodes-library/nodes/Discord/README.md`. To reach the future Start node readme in `docs/nodes-library/core-nodes/Start/README.md`, you would have to go back up to `nodes-library` and then down into `/core-nodes/Start/README.md`.

File: docs/nodes/nodes-library/nodes/Discord/README.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+# Discord
+
+Discord is a voice, video, and text communications platform for groups. Discord allows users to programatically send messages using webhooks.
```
Comment (ianjennings) [inline docs/nodes/nodes-library/nodes/Discord/README.md]: Add link to discord.gg or whatever website.

File: docs/nodes/nodes-library/nodes/Discord/README.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+### 1. Start node
+
+The start node exists by default when you create a new workflow.
+
+### Discord Node
+
+1. In order to create a Discord webhook that sends content to your channel, go to the settings of your channel.
+2. Click Webhooks.
+3. Name your bot and copy the link of the webhook.
+4. Paste your webhook into the *Webhook URL* field of the Discord Node.
```
Comment (ianjennings) [inline docs/nodes/nodes-library/nodes/Discord/README.md]: Maybe say 'on n8n"

File: docs/nodes/nodes-library/nodes/Discord/README.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+5. Enter your message in the *Text* field.
+6. Click on *Execute Node* to run the workflow. :tada:
+
+![GIF of Webhooks](https://media.giphy.com/media/WtI5wgNkgw0YoNqhfu/giphy.gif)
```
Comment (ianjennings) [inline docs/nodes/nodes-library/nodes/Discord/README.md:34]: I usually default to putting these above instructions.

File: docs/nodes/nodes-library/nodes/Discord/README.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+
+The Discord Node does not require Authentication, but you must have access to a channel's settings to use webhooks.
+
+## Basic Operations
+
+- Send messages in a Discord Channel
+
+## Example Usage
+
+This workflow allows you to send a message to a Discord channel using webhooks. You can also find the [workflow](https://n8n.io/integrations/n8n-nodes-base.discord) on this website. This example usage workflow would use the following two nodes.
```
Comment (ianjennings) [inline docs/nodes/nodes-library/nodes/Discord/README.md]: This is generally how "call to actions" are structured:

```md
You can also [find the workflow on this website](https://n8n.io/integrations/n8n-nodes-base.discord).
```

**Review by ianjennings — CHANGES_REQUESTED** [review]

File: docs/nodes/nodes-library/nodes/Discord/README.md (hunk 3 above)
Comment (erin2722) [reply docs/nodes/nodes-library/nodes/Discord/README.md]: You're Right! Fixed it!

File: docs/nodes/nodes-library/nodes/Discord/README.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+The Discord Node does not require Authentication, but you must have access to a channel's settings to use webhooks.
+
+## Basic Operations
+
+- Send messages in a Discord Channel
+
+## Example Usage
+
+This workflow allows you to send a message to a Discord channel using webhooks. You can also [find the workflow on this website](https://n8n.io/integrations/n8n-nodes-base.discord). This example usage workflow would use the following two nodes.
+- [Start](../../core-nodes/Start)
```
Comment (ivov) [inline docs/nodes/nodes-library/nodes/Discord/README.md:14]: That link points to a dir. I think it should be `(../../core-nodes/Start/README.md)`

Comment (erin2722) [reply docs/nodes/nodes-library/nodes/Discord/README.md:14]: Same here, I the dir works as well as the file.

**Pull request merged**